More documentation fixes (#75)
epinio/docs: src/howtos/install_epinio_on_rancher.md

If you do not have such, please use the Rancher's [installation guides](https://rancher.com/docs) to set one up.

> This page was written using the following versions:

Reviewer: ```suggestion
This how-to was written using the following versions:
```

* [epinio helm chart 0.7.1](https://github.com/epinio/helm-charts/releases/tag/epinio-0.7.1)
* Rancher 2.6.3

## Kubernetes cluster

The two supported methods are to either deploy a new cluster or to import an existing one into Rancher. The steps to take for either method are detailed in the Rancher documentation (See `Cluster Provisioning`).</br>
*While it is possible (for development) to deploy Epinio on the same node where Rancher is installed, this requires more steps and is not supported by Rancher either.*

### Rancher settings

At the time of writing, Epinio is not included in the latest stable Rancher version.</br>
This means that you have to update two settings in your Rancher dashboard.</br>
Go to `Global settings` > `Advanced settings`  and update the following settings:</br>

| | |
|--|--|
| `ui-dashboard-index`  | `https://releases.rancher.com/dashboard/epinio-v0.6.0/index.html` |
| `ui-offline-preferred`  | `Remote` |

> *Please, keep in mind that `ui-dashboard-index` value will change over the time until it is merged in stable Rancher version.

Reviewer: ```suggestion
> NOTE: Please keep in mind that `ui-dashboard-index` value will change over time until it's merged in stable Rancher version.
```

## Epinio Prerequisites

### Install an Ingress controller

> If there is no Ingress controller running yet on the cluster, install Traefik.

Reviewer: ```suggestion
If there's no Ingress controller running yet on the cluster, you need to install one. In the example below, Traefik will be installed.
```

Reviewer: The Traefik installation link might need to be updated/standardized if we explain it somewhere else.

Author: In that case, I think I can put this link: https://docs.epinio.io/installation/installation.html#ingress-controller

> NOTE: If you are in the rancher dashboard in the browser, most likely you already have an Ingress controller
> running.

Go to `Apps & Marketplace` >  `Repositories` > `Create`.

Give a name to the repo and use `https://helm.traefik.io/traefik` as the index URL.

Once done, go to the "Charts" menu and install the `traefik` chart.
See also the [Install Epinio](../installation/installation.html#ingress-controller) page for some helm values you can set.

### Install cert-manager

> If cert manager is not installed yet you can install it by adding the relevant repository

Reviewer: ```suggestion
If cert manager is not installed yet, you can install it by adding the relevant repository
```

Give a name to the repo and use `https://epinio.github.io/helm-charts` as index URL.

Once done, go to the "Charts" menu and install the `epinio` chart.

> While you can select the version you want most of the time you should choose the latest one (default). This page was written with version `0.7.1`. Then click install in the top right corner.

Reviewer: ```suggestion
> NOTE: While you can select the version you want, it's strongly advised you choose the latest one (default). This how-to was written with version `0.7.1`. 
```

> On the next screen, you can select a namespace where the deployment will happen. You also have to set a name for the deployment.

Reviewer: ```suggestion
Click install in the top right corner.

On the next screen, you can select a namespace where the deployment will happen. You also have to set a name for the deployment.
```

The following screen is where you can configure Epinio according to your needs.</br>
The 2 most important settings here are:
- `Domain`: [Explained here](../installation/dns_setup.md)
- `Access control allow origin`:  It is mandatory to insert your Rancher dashboard URL into this field. It basically allows Rancher to talk with the Epinio API.

Once everything is configured, click on the install button and then wait a few minutes for the installation to complete.</br>
After that follow the instructions given by the Helm install command output.

## Access the Epinio menu

> Click on the "burger" menu icon at the top left and then the Epinio icon under **GLOBAL APPS**.

Reviewer: ```suggestion
Click on the "burger" menu icon (☰) at the top left and then the Epinio icon under **GLOBAL APPS**.
```

You should see your fresh Epinio instance there.

> __Attention__. If you use an untrusted certificate, you have to click on the URL, accept the security exception and use the refresh button shown below the page title, i.e. __Epinio instances__.</br>

Reviewer: ```suggestion
> ATTENTION: If you use an untrusted certificate, you have to click on the URL, accept the security exception and use the refresh button shown below the page title, i.e. __Epinio instances__.
```
